script.js

```diff
@@ -1,4 +1,67 @@
+// User inputs one of "scissors", "paper", or "stone",
+// Program internally randomly chooses scissors, paper, or stone,
+// Program outputs whether the user won, the program won, or it's a draw.
+// Rules: scissors beats paper, paper beats stone, and stone beats scissors.
+// If both parties choose the same object, it's a draw.
+
+// reverse scissors paper stone game
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var computer = generateScissorsPaperStone();
```

Comment:
Code works top-down. For future activities, declare the function first before calling it.